src/mixins/GridConfig.js

```diff
@@ -36,7 +36,7 @@ export default {
 		getGridConfig.$on('changed', val => {
 			this.gridConfig = val
 		})
-		console.debug('Current grid config', getGridConfig.gridConfig)
+		// console.debug('Current grid config', getGridConfig.gridConfig)
```

Comment:
hum?

Reply:
this log annoy me, but you left it here for long :joy: 
i just wanted to sacralize it by commented it :joy:

Reply:
Nah, it's here on purpose, I tend to leave debug logs
You can hide them in the console (hidden by default btw) ;)